From 466ab4e0a1fe0536ed5b4c5ef7367574e34745dd Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Sat, 14 Dec 2013 16:18:50 +0000
Subject: [PATCH] builder: tests: Add test-virt-builder-planner.sh to
 EXTRA_DIST.

This fixes commit a27963b67ba1140c29c145a1ce14dbfc05f8d7da.

(cherry picked from commit 6901cd2250fc25c1761f191fb02db74fecf4703d)
---
 builder/Makefile.am | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/builder/Makefile.am b/builder/Makefile.am
index 11967e9..3f35cc6 100644
--- a/builder/Makefile.am
+++ b/builder/Makefile.am
@@ -30,7 +30,8 @@ EXTRA_DIST = \
 	virt-builder.pod \
 	virt-index-validate.pod \
 	test-virt-builder.sh \
-	test-virt-builder-list.sh
+	test-virt-builder-list.sh \
+	test-virt-builder-planner.sh
 
 CLEANFILES = *~ *.cmi *.cmo *.cmx *.cmxa *.o virt-builder
 
-- 
1.8.4.2

